mandatory/2-function-creation.js

```diff
@@ -15,26 +22,69 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200) {
+    return totalPrice * 0.9;
+  } else {
+    return totalPrice * 0.95;
+  }
+}
+
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let num = 1;
+  
+  while (num <= limit) {
+    if (num % 2 !== 0) {
+      console.log(num);
+    }
+    num++;
+  }
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
+function buyTwoGetTheCheapestFree(price1, price2) {
+  if (price1 <= 0 || price2 <= 0) {
+    return "Invalid prices";
+  }
+
+  let totalPrice = price1 + price2;
```

Comment:
This is a very complicated way of doing this. Could you think of a simpler way that doesn't involve adding an subtracting?

Reply:
I just realized it was really complicated.  

function buyTwoGetTheCheapestFree(price1, price2) {
  if (price1 > price2) {
    return price1;
  } else {
    return price2;
  }
}